spec/routemaster/drain/caching_spec.rb

```diff
@@ -37,7 +38,7 @@
   it 'increments the event index' do
     ei_double = double(increment: 1)
     allow(Routemaster::EventIndex).to receive(:new).and_return(ei_double)
-    expect(ei_double).to receive(:increment).exactly(3).times
+    expect(ei_double).to receive(:increment).exactly(4).times
```

Comment:
Previously we didn't expire for every event (as we did it after the dirty map) - this removed one of the duplicate events from the spec. Now we expire for every event (but still initiate a de-duped cache step)